src/lib/src/header/header.component.css

```diff
@@ -34,7 +34,7 @@ a.drawer {
   width: 20px;
   margin: 0 14px;
   line-height: 50px;
-  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAQAQMAAAAs1s1YAAAABlBMVEU2iqL///9rzgLQAAAAAnRSTlP/AOW3MEoAAAASSURBVAjXY2BAA///fwBjIsUBd4sLub+XcZUAAAAASUVORK5CYII=");
+  background-image: url("ngx-prx-styleguide/assets/images/ic_header_menu.png");
```

Comment:
yeah I couldn't really figure that out because it's postcss at that point that needs the path not typescript. I played with the base path config for the postcss plugin but couldn't get it because ngx-prx-styleguide part of the path just doesn't exist and it doesnt have anything to support pseudo path from what I could tell.